src/main/scala/no/ndla/imageapi/controller/ImageController.scala

```diff
@@ -85,5 +108,30 @@ trait ImageController {
         case None => halt(status = 404, body = Error(NOT_FOUND, s"Image with id $imageId not found"))
       }
     }
+
+    get("/full/:filename", operation(getImageFile)) {
```

Comment:
Ah. Riktig. Selv om det ikke er helkrise, gitt at vi cacher dataene er det jo dust å svinge innom Scala når vi ikke må. Vi _kunne_ dyttet id'ene ut i S3, men jeg har ikke egentlig lyst til det for jeg tror det fort kan bite oss i baken senere.

Jeg liker ikke `/image/api/v1/image/<filnavn>` så godt fordi forskjellen mellom `.../image/...` og `.../images/...` blir så liten, og lett å ta feil av. Hva da med `/image/api/v1/content/<filnavn>` el.n.s.?